packages/api/src/middleware/authentication.ts

```diff
@@ -228,9 +228,33 @@ const extractUserEmailFromAuthToken = async (
   next();
 };
 
+const extractUserIsAdminFromAuthToken = async (
+  req: Request<unknown, unknown, { admin?: boolean }>,
+  _: Response,
+  next: NextFunction
+): Promise<void> => {
+  const authenticationToken = getAuthTokenFromRequest(req);
+  // NOT sure if this try-catch is the way to go... I tried to copy extractUserEmailFromAuthToken
+  // and use FUSIONAUTH_ADMIN_APPLICATION_ID instead of FUSIONAUTH_BACKEND_APPLICATION_ID, but I get a timeout
+  try {
+    const email = await getOptionalValueFromAuthToken(
+      authenticationToken,
+      emailKey,
+      process.env["FUSIONAUTH_ADMIN_APPLICATION_ID"] ?? ""
+    );
+    if (email !== "") {
+      req.body.admin = true;
+    }
+    next();
+  } catch (innerErr) {
+  }
```

Comment:
This empty catch doesn't seem like the right path here; in almost any case when an exception occurs we at least want to know about it. If this is what fixed the timeout error, the error is presumably still happening, it just gets swallowed by this catch now.

Since what you're doing here is just about the same as what `extractUserEmailFromAuthToken` does, I suspect what's going on is that your FusionAuth env vars aren't set to the correct values; indeed it looks like they couldn't be, because you don't have access to FusionAuth yet. I'll set up your account there now.

Reply:
You should have an email inviting you to FusionAuth; once you get into their admin console you should be able to find the values you need for the environment variables.